opensearch/src/main/java/org/opensearch/sql/opensearch/storage/OpenSearchIndexScan.java

```diff
@@ -158,6 +160,22 @@ public void pushDownLimit(Integer limit, Integer offset) {
     sourceBuilder.from(offset).size(limit);
   }
 
+  /**
+   * Add highlight to DSL requests.
+   * @param field name of the field to highlight
+   */
+  public void pushDownHighlight(String field) {
+    SearchSourceBuilder sourceBuilder = request.getSourceBuilder();
+    if (sourceBuilder.highlighter() != null) {
+      sourceBuilder.highlighter().field(StringUtils.unquoteText(field));
+      sourceBuilder.highlighter(sourceBuilder.highlighter());
```

Comment:
is this unnecessary?

Reply:
good catch, i believe this can be removed.